app/views/customers/sessions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="sigin-container">
+  <div class="row text-center">
+    <%= image_tag "sendaguy.png", width: '60%' %> 
```

Comment:
That's very minor comment: if you use double quotes everywhere in ruby code (as I see it in this PR), let's use it here too ` width: "60%"`.